data/outfits.txt

```diff
@@ -301,6 +301,16 @@ outfit "Small Bunk Room"
 	unplunderable 1
 	description "Captains who do not care particularly much about the comfort of their passengers or crew can usually find a way to shove a bunk bed in behind the power generator or to hang a few hammocks from the engines as a makeshift way to make space for more people."
 
+outfit "Luxury Accommodations"
+	category "Systems"
+	cost 250000
+	thumbnail "outfit/luxury accommodations"
+	"mass" 10
+	"outfit space" -10
+	"required crew" 2
+	"unplunderable" 1
+	description "Most deep space voyages involve a certain amount of passenger discomfort. With the addition of a few extra fresh water tanks, private bathrooms, wooden fixtures, and some hospitality staff, any ship can be made attractive to a significantly higher--and better paying--class of passenger."
```

Comment:
`higher - and better paying - class`

Reply:
I think the em-dash is grammatically correct here?

Reply:
https://github.com/endless-sky/endless-sky/pull/2221#discussion_r109287047

It's happened before -- MZ isn't a fan of double dashes.

(I'm kind of hoping he reads that and chuckles to himself.)

Reply:
Well, the difference here is that there aren't spaces on each end of the em dashes.

I can kind of understand not liking the double dashes version; there's just something very off-putting about it.

I feel like Michael himself used it somewhere, though. I don't remember where, and a search brings up nothing.

Reply:
https://github.com/endless-sky/endless-sky/pull/2395/files

You're thinking of the Kestrel description, which I changed based on his comment and got merged.

Reply:
Ah, I was just looking there. I thought something was wrong.